mqemitter-redis.js

```diff
@@ -14,7 +14,8 @@ function MQEmitterRedis (opts) {
     return new MQEmitterRedis(opts)
   }
 
-  opts = opts || {}
+  opts = opts || { showFriendlyErrorStack: true }
```

Comment:
why this change?

Reply:
Sorry, missed it, was to identify where the unhandled rejection was coming from.